### PR TITLE
Speed up docker builds

### PR DESCRIPTION
**Pull request by benaadams**
## Changes

- Don't publish ReadyToRun for faster deployment
- Nethermind is a very long running process so initial startup time isn't so important vs better instrumentation from tier0 to produce improved teir1 code via Jit PGO

#### What types of changes does your code introduce?

- [x] Optimization
- [x] Build-related changes

#### Requires testing

- [x] No

### CODE REVIEW

**Review by rubo — CHANGES_REQUESTED** [review]
We should not disable R2R for Docker only. Nethermind builds should not differ depending on distribution channels. If we consider R2R not important, we disable it in general.
The long build times for the Docker image are due to GitHub Actions runners. We can either switch to a more powerful one (~8 mins) or just wait a little more with the current one. It's not a big deal.

**Comment by benaadams** [comment]
> We should not disable R2R for Docker only. Nethermind builds should not differ depending on distribution channels. If we consider R2R not important, we disable it in general. The long build times for the Docker image are due to GitHub Actions runners. We can either switch to a more powerful one (~8 mins) or just wait a little more with the current one. It's not a big deal.

I did disable it everywhere?

**Comment by rubo** [comment]
> I did disable it everywhere?

Then why have it `false` or `0` explicitly?

**Comment by benaadams** [comment]
> > I did disable it everywhere?
> 
> Then why have it `false` or `0` explicitly?

It only applies to publishing; so changed it to `false` in .csproj

The other one in docker files turns off ReadyToRun for the framework so it Jits everything but that can only be set via environment var not any project or build setting :(

**Comment by rubo** [comment]
> The other one in docker files

Do we need it? Isn't it off by default?

File: src/Nethermind/Nethermind.Runner/Dockerfile [hunk 1]
```diff
@@ -23,4 +23,5 @@ RUN dotnet publish Nethermind.Runner.csproj -c release -o /app/publish -p:UseApp
 FROM base AS final
 WORKDIR /app
 COPY --from=publish /app/publish .
+
```
Comment (rubo) [inline src/Nethermind/Nethermind.Runner/Dockerfile:26]:
```suggestion
```

File: src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <InvariantGlobalization>true</InvariantGlobalization>
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <TieredPGO>true</TieredPGO>
-    <PublishReadyToRun>true</PublishReadyToRun>
+    <PublishReadyToRun>false</PublishReadyToRun>
```
Comment (rubo) [inline src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj:12]: Maybe it's better just to delete the line.

```suggestion
```

Comment (benaadams) [reply src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj]: sure

**Pull request merged**